src/media/css/app-list.styl

```diff
@@ -110,16 +110,31 @@ section.langpacks > div {
     }
     .app-list-app {
         border-bottom: $border;
-        border-right: $border;
         padding: $desktop-tile-padding;
         width: 50%;
 
+        +ltr() {
+            border-right: $border;
+        }
         &:nth-child(2n) {
             border-right: 0;
```

Comment:
I'd rather see `border-right: 0` in the `+ltr()` block rather than remove the border and add it back in `+rtl()`.